e2e_example/lib/transformer.dart

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
-import 'package:build/build.dart';
-
-import 'copy_builder.dart';
-
-class CopyTransformer extends BuilderTransformer {
-  @override
-  final List<Builder> builders = [new CopyBuilder()];
-
-  CopyTransformer.asPlugin(_);
```

Comment:
I didn't realize that asPlugin had extra meaning which made this useful. Restored it.